style.css

```diff
@@ -212,23 +218,22 @@ header {
 }
 
 .action .btn:hover {
-  background: ##6070FF;
+  background-color: #6070ff;
   color: #fff;
-  transition: all 0.8s ease;
 }
 
 .action .btn:enabled {
-  background: #fff;
+  background-color: #fff;
   color: #6070ff;
 }
 
 .action .btn:disabled {
-  background: #fff;
+  background-color: #fff;
   color: #5e6c84;
 }
 
 .action .btn:active {
-  background: #2230d2;
+  background-color: #2230d2;
   color: #fff;
 }
```

Comment:
- [ ] Hovering is not working properly on the desktop version. you can try to add the `:hover` and `:active` with the `btn:enabled` properties so you end up with fewer lines of code:
`.action .btn:disabled `
`.action .btn:enabled:hover `
`.action .btn:enabled:active `